fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_daylight.dm

```diff
@@ -97,6 +105,7 @@
 				M.current.playsound_local(null, 'sound/ambience/ambimystery.ogg', 100, 1)
 			else if(danger_level == 5)
 				M.current.playsound_local(null, 'sound/spookoween/ghosty_wind.ogg', 90, 1)
+			return
```

Comment:
This is the line causing vassals to not get warned. Why was a return added here?

Reply:
for(var/datum/mind/M as anything in get_antag_minds(/datum/antagonist/bloodsucker))

Vassals arent bloodsuckers, are they?

Reply:
Return exits the proc, so as soon as the first bloodsucker is warned the warn_daylight proc ends before it warns the rest of the bloodsuckers or any of the vassals in the proceeding for loops

Reply:
The intent was that Favorite Vassal bloodsuckers wouldnt get both warnings.